src/hw9/Main.java

```diff
@@ -3,17 +3,17 @@
 import static hw9.ArrayValueCalculator.doCalc;
 
 public class Main{
-    public static void main(String[] args) {
+    public static void main (String[] args){
 
         String[][] arr = {{"1", "2", "3", "4"}, {"5", "6", "7", "8"}, {"9", "10", "11", "12"}, {"13", "14", "15", "16"}};
 
         try {
-            int result = doCalc(arr);
-            System.out.println("The sum of the values in the array is: " + result);
+            int result = doCalc (arr);
```

Comment:
Formatting `int result = doCalc (arr);` should be as follows: `int result = doCalc(arr);`